SRC/NETWORK/sntp.c

```diff
@@ -1,19 +1,23 @@
 /* SNTP client for Ethernet UNAPI
    By Konamiman 2/2010
+   v1.1 by Oduvaldo Pavan Junior ( ducasp@gmail.com )
    
+   v1.1 fixes when the count of seconds will end with HH:MM:60 when 
+   it should be HH:MM+1:00
+
    Compilation command line:
-   
+
    sdcc --code-loc 0x170 --data-loc 0 -mz80 --disable-warning 196
         --no-std-crt0 crt0_msxdos_advanced.rel msxchar.rel asm.lib sntp.c
    hex2bin -e com sntp.ihx
    
    ASM.LIB, MSXCHAR.REL and crt0msx_msxdos_advanced.rel
    are available at www.konamiman.com
-   
+
    (You don't need MSXCHAR.LIB if you manage to put proper PUTCHAR.REL,
    GETCHAR.REL and PRINTF.REL in the standard Z80.LIB... I couldn't manage to
    do it, I get a "Library not created with SDCCLIB" error)
-   

```

Comment:
It would be great to update the compilation instruction while we're at it. Please change them to this:

```
Compilation command line:

sdcc --code-loc 0x180 --data-loc 0 -mz80 --disable-warning 196 --no-std-crt0 crt0msx_msxdos_advanced.rel printf_simple.rel putchar_msxdos.rel asm.lib sntp.c

crt0msx_msxdos_advanced.rel, printf_simple.rel, putchar_msxdos.rel and asm.lib are available at www.konamiman.com
```

Also note that the _You don't need MSXCHAR.LIB...` comment is outdated and it must be removed.

Reply:
I've updated it to reflect the new build instructions. :-)